shotover-proxy/Cargo.toml

```diff
@@ -92,6 +93,7 @@ reqwest = "0.11.6"
 metrics-util = "0.14.0"
 cdrs-tokio = { git = "https://github.com/krojew/cdrs-tokio" }
 scylla = { version = "0.4.7", features = ["ssl"] }
+rstest = "0.15.0"
```

Comment:
is this unused?

Reply:
Oh sorry, searching `rstest` didnt turn up any results, I must have failed to expand the hidden file it was in.
:+1: